webapp/app/user/userManagement/edit/countrySelectionModal.tsx

```diff
@@ -2,8 +2,14 @@ import React from 'react'
 import * as R from 'ramda'
 import { Modal, ModalBody, ModalClose, ModalFooter, ModalHeader } from '@webapp/components/modal'
 import { useI18n } from '@webapp/components/hooks'
+import { useSecondaryGroupedRegions } from '@webapp/store/app/hooks'
+
+
+type Props = any
```

Comment:
why props any?

Reply:
This component is deprecated but still in use